MyLibrary/MyLibraryApp.Application/Services/OrderService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using MyLibraryApp.Application.Models.Orders;
+using MyLibraryApp.Application.Services.Interfaces;
+using MyLibraryApp.DataAccess.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.Application.Services
+{
+    public class OrderService: IOrderService
+    {
+        private readonly IOrderRepository repository;
+        public OrderService(IOrderRepository repository)
+        {
+            this.repository = repository;
+        }
+
+        public async Task<OrderResponseModel> CreateOrderAsync(CreateOrderRequestModel requestModel)
+        {
+            var order = requestModel.ToOrder();
+            await repository.AddAsync(order);
+            return OrderResponseModel.FromOrder(order);
+        }
+
+        public async Task UpdateOrderAsync(UpdateOrderRequestModel requestModel)
+        {
+            var order = requestModel.ToOrder();
+            await repository.UpdateAsync(order);
+        }
+        
+
+        public async Task<OrderResponseModel> GetOrderByIdAsync(int id)
```

Comment:
All get by id methods should be updated to handle the scenario in which the entity does not exist in the database and throw an exception.

Same with update and delete - if the entity does not exist, how can it be updated/deleted?